src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -17,7 +17,15 @@ const ReleaseCommit = React.createClass({
     repository: React.PropTypes.object,
 
   },
+
+  getCommitUrl() {
+    if (this.props.repository.provider.id === 'github') {
```

Comment:
yeah, moving this so it can be defined by the plugins is on my list of things to do and i added a todo.